recipes/qd/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.3.17" %}
+
+package:
+  name: qd
+  version: {{ version }}
+
+source:
+  fn: qd-{{ version }}.tar.gz
+  url: http://crd.lbl.gov/~dhbailey/mpdist/qd-{{ version }}.tar.gz
+  sha256: c58e276f6fcf5f2f442c525f3de42ea00004734572b29c74028bbda0ad81096d
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - gcc
+  run:
+    - libgcc
```

Comment:
It is apparently also linking to libquadmath which I believe is included with libgcc?